keanu-python/keanu/vartypes.py

```diff
@@ -16,7 +16,7 @@
 pandas_types = Union[Series, DataFrame]
 numpy_types = Union[ndarray]
 
-tensor_arg_types = Union[primitive_types, pandas_types, numpy_types]
+tensor_arg_types = Union[primitive_types, pandas_types, numpy_types, JavaObjectWrapper, str]
```

Comment:
What's going on here? How/when would one coerce a `JavaObjectWrapper` or string into a tensor?

Reply:
This is basically here to try and convince mypy to behave tbh in the least impactful way possible.  I'll chat to you offline.

Reply:
Resolved in a new way (by adding a bit more code) so that it's hopefully a lot cleaner now.  Let me know what you think.